acceptance/suites/tests/010-puppetserver-cli/subcommand/foreground.rb

```diff
@@ -0,0 +1,44 @@
+test_name "Puppetserver 'foreground' subcommand tests."
+
+# --------------------------------------------------------------------------
+# This behavior needs to be baked into Beaker but is not yet. SERVER-79 tracks
+# the need to fix this once Beaker has been modified to make the paths to these
+# commands available.
+#
+if master.is_pe?
+  cli = "pe-puppetserver"
+else
+  cli = "puppetserver"
+end
+# --------------------------------------------------------------------------
+
+# puppetserver seems to take about 45s to start up
+timout_length = "60s"
```

Comment:
Yeah, I was thinking the same thing. Puppet is installed just prior to this being run, and the script shuts it down first, so yeah I think it's reasonable to assume that it will always have been run at least once